deploy/ssh.sh

```diff
@@ -102,6 +99,19 @@ ssh_deploy() {
     _cleardomainconf Le_Deploy_ssh_multi_call
   fi
 
+  _deploy_ssh_servers=$Le_Deploy_ssh_server
+  for Le_Deploy_ssh_server in $_deploy_ssh_servers
+  do
```

Comment:
Travis wants the `do` on the same line as the `for` (preceded by a semicolon of course)